packages/patternfly-4/react-styles/src/build/util.js

```diff
@@ -40,7 +40,7 @@ export function writeCSSJSFile(rootPath, originalPath, destinationPath, contents
 
 export function getRelativeImportPath(from, to) {
   const parsedTo = path.parse(to);
-  const newImportPath = path.normalize(path.join(relative(from, parsedTo.dir), parsedTo.base).replace(/\\/g, ''));
```

Comment:
Not quite sure why we've removed backslashes here. From what I can see, it didn't have any effect on Linux/OS X, and completely broke paths on Windows.

Am I missing something?

Reply:
i'm honestly not sure... but just ensuring you can run `yarn clean; yarn build; yarn start:pf4` and we verifying we still have styles loading correctly should hopefully verify this...

Reply:
(rebuilding styles)